packages/backend/src/auth/users/user.entity.ts

```diff
@@ -1,15 +1,31 @@
-import {Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, BaseEntity, Unique } from 'typeorm';
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  BaseEntity,
+  Unique,
+  OneToMany,
```

Comment:
wylacz automatyczne formatowanie bo jestes podpisany przez to potem nawet w javowych androidowych plikach